dart/test/protocol/breadcrumb_test.dart

```diff
@@ -103,17 +103,43 @@ void main() {
           'method': 'GET',
           'status_code': 200,
           'reason': 'OK',
-          'duration': '0:00:00.000000',
+          'duration': '0:00:00.055000',
```

Comment:
isnt' duration an int?

@romtsn @brustolin ?

Reply:
it's not rrweb crumb but sentry crumb here, so the data bag is arbitrary

Reply:
yes, this is not getting to rrweb